### PR TITLE
Update development dependencies

### PR DESCRIPTION
**Pull request by bertdeblock**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,24 +43,22 @@
     "walk-sync": "^2.2.0"
   },
   "devDependencies": {
-    "chai": "^4.3.4",
+    "@release-it-plugins/lerna-changelog": "^6.1.0",
+    "chai": "^4.4.1",
```
Comment (bertdeblock) [inline package.json:47]: Latest v4, because v5 is pure ESM.

File: package.json [hunk 2]
```diff
@@ -43,24 +43,22 @@
...
-    "glob": "^7.2.0",
-    "json": "^10.0.0",
-    "mocha": "^8.2.1",
+    "eslint-plugin-prettier": "^5.1.3",
+    "mocha": "^10.3.0",
     "mockery": "^2.1.0",
     "nyc": "^15.1.0",
-    "prettier": "^2.4.1",
-    "release-it": "^14.11.6",
-    "release-it-lerna-changelog": "^3.1.0",
```
Comment (bertdeblock) [inline package.json:61]: Renamed to `@release-it-plugins/lerna-changelog`.

File: package.json [hunk 3]
```diff
@@ -43,24 +43,22 @@
...
     "codecov": "^3.8.3",
-    "ember-cli": "~3.22.0",
-    "eslint": "^7.31.0",
-    "eslint-config-prettier": "^6.15.0",
+    "ember-cli": "~5.7.0",
+    "eslint": "^8.57.0",
+    "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^3.1.4",
-    "glob": "^7.2.0",
```
Comment (bertdeblock) [inline package.json:54]: Seems unused.

File: package.json [hunk 4]
```diff
@@ -43,24 +43,22 @@
...
-    "json": "^10.0.0",
```
Comment (bertdeblock) [inline package.json:55]: Seems unused.

**Review by kategengler — DISMISSED** [review]

**Comment by bertdeblock** [comment]
Closing, due to lack of time.

File: .github/workflows/ci.yml [hunk 5]
```diff
@@ -31,20 +31,21 @@ jobs:
...
       - name: Install Dependencies
         run: yarn install --frozen-lockfile
       - name: Run Tests
-        run: yarn node-test-with-coverage
-      - uses: codecov/codecov-action@v4
-        with:
-          fail_ci_if_error: true
-          token: ${{ secrets.CODECOV_TOKEN }}
+        run: yarn node-test
+      #   run: yarn node-test-with-coverage
```
Comment (kategengler) [inline .github/workflows/ci.yml:44]: I think enable this but put `if: github.event_name == 'push'`

And a comment with this link https://github.com/codecov/codecov-action/issues/1580 so we know to re-enable

Comment (kategengler) [reply .github/workflows/ci.yml:44]: It looks like codecov-action@v5 should now work per that linked issue

Comment (bertdeblock) [reply .github/workflows/ci.yml]: Indeed, but I forgot to push my last commit :smile: Should be okay now.

**Review by kategengler — APPROVED** [review]

**Comment by kategengler** [comment]
Thank you!

**Pull request merged**